api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -191,6 +194,9 @@ message TlsSessionTicketKeys {
       [(validate.rules).repeated = {min_items: 1}, (udpa.annotations.sensitive) = true];
 }
 
+// All *DataSource*s in this message support *WatchedFile* when
```

Comment:
Same as above: `crl` DataSource is not watched & reloaded today, but is the intention to add that now?